### PR TITLE
full audit

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by WilliamLoy** [comment]
@cyrichardson this one is ready for editing :)

**Comment by cyrichardson** [comment]
Squeezing it in now before I go!

**Comment by cyrichardson** [comment]
@stephamon Please be my backup on this one for editing. I'm running out of time today (and I'm not allowed to go over 20 hours). @WilliamLoy FYI

**Comment by rackernexus** [comment]
Build 1092 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-60b6b7337/how-to/). valid for 14 days

**Comment by rackernexus** [comment]
Build 1094 was successful. Staging content [Preview!](https://support-revs.staging.deconst.org/build-c90460425/how-to/). valid for 14 days

File: content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md [hunk 1]
```diff
@@ -1,121 +1,109 @@
...
 product: Cloud Servers
 product_url: cloud-servers
 ---
 
-**Warning:** Email in the Cloud (Rackspace or otherwise), even with
-today's vast standards, is considered to be "best effort" without any
-guarantees for reliability or deliverability. Follow the steps below to
-have a better email sending reputation. As an additional precaution, it
-is helpful to research and consider using the following:
+**Warning:** Managing a mail server in the cloud provides no guarantee email will be delivered to recipients' Inboxes. Establishing and maintaining email deliverability is complex undertaking. See our [hosted email offerings](https://support.rackspace.com/how-to/#cloud-office) if you are in need of a managed email solution.
```
Comment (cyrichardson) [inline content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md]: email is delivered to   (avoid future tense)
is a complex undertaking   (add article "a")
if you need a managed   (simply wording)

File: content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md [hunk 2]
```diff
@@ -1,121 +1,109 @@
...
 ---
 
-**Warning:** Email in the Cloud (Rackspace or otherwise), even with
-today's vast standards, is considered to be "best effort" without any
-guarantees for reliability or deliverability. Follow the steps below to
-have a better email sending reputation. As an additional precaution, it
-is helpful to research and consider using the following:
+**Warning:** Managing a mail server in the cloud provides no guarantee email will be delivered to recipients' Inboxes. Establishing and maintaining email deliverability is complex undertaking. See our [hosted email offerings](https://support.rackspace.com/how-to/#cloud-office) if you are in need of a managed email solution.  
+
+To improve your email server's sending reputation, it is helpful to research and consider using the following solutions:
```
Comment (cyrichardson) [inline content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md]: To improve your email server's sending reputation, you need to research   (dangling modifier)

File: content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md [hunk 3]
```diff
@@ -1,121 +1,109 @@
...
 -   [DomainKeys Identified Mail (DKIM)](/how-to/rackspace-cloud-dns-additional-resources)
 -   [Domain-based Message Authentication, Reporting & Conformance (DMARC) DNS records](http://www.dmarc.org/index.html)
 
-It is vital that the Cloud Server has the basics such as the hostname
-and Reverse DNS correctly set before we configure any sort of mail
-service. This article looks at the hostname and reverse DNS (RDNS)
-settings on the Cloud Server.
-
-### Assumptions
+It is vital that your Cloud Server has the hostname and Reverse Domain Name System (RDNS) correctly set before you configure any sort of mail service. This article reviews the hostname and RDNS settings necessary for a Cloud Server being configured for email.
```
Comment (cyrichardson) [inline content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md]: Before you configure any sort of mail service, it is vital that your Cloud Server has the hostname and Reverse Domain Name System (RDNS) correctly set.

File: content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md [hunk 4]
```diff
@@ -1,121 +1,109 @@
...
 
-The only assumptions made for these email articles are:
+### Prerequisites
 
--  You are running and have set up Ubuntu Hardy (The instructions may work on other distros 
-   but it has not been tested and is not guaranteed).
+The instructions in this article utilize the following technology:
 
-### Configure hostname
+-  Ubuntu Hardy operating system. (The instructions may work on other Linux&reg;
```
Comment (cyrichardson) [inline content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md]: The Ubuntu&reg; Hardy operation system   (Waldo project requirement)
The instructions might work

File: content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md [hunk 5]
```diff
@@ -1,121 +1,109 @@
...
+   distributions but it has not been tested and is not guaranteed).
 
-The first thing to look at is the Cloud Server hostname.
+### Configure the hostname
 
-This is simply the name of the Cloud Server and is used in the headers
+A hostname is the name of the Cloud Server and is used in the headers
```
Comment (cyrichardson) [inline content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md]: A hostname is the name of the Cloud Server. The hostname is used in the headers of the email. You can think of the headers as the address and sender label on the email.

File: content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md [hunk 6]
```diff
@@ -1,121 +1,109 @@
...
+  **Note:** These instructions use the domain **mail.democloud.com**.
 
-    sudo nano /etc/hostname
+2. Change the hostname to match the domain:
 
-Replace the current hostname (cloudserver1) with the one you need (in this case **mail.democloud.com**).
+    `sudo nano /etc/hostname`
 
-The second file to edit is:
+    Replace the current hostname `cloudserver1` with your selected domain name which in this example is **mail.democloud.com**.
```
Comment (cyrichardson) [inline content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md]: name, which

File: content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md [hunk 7]
```diff
@@ -1,121 +1,109 @@
...
-    127.0.0.1       cloudserver1
+    The default looks like this on our example cloud server:
 
-Following on from what we are doing, replace 'cloudserver1' with
-**mail.democloud.com**.
+    `127.0.0.1       localhost localhost.localdomain`
+    `127.0.0.1       cloudserver1`
 
-Of course, replace mail.democloud.com with your domain.
+    Replace `cloudserver1` with `mail.democloud.com`.On your Cloud Server you should replace `mail.democloud.com` with your domain.
```
Comment (cyrichardson) [inline content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md]: `mail.democloud.com`. On your Cloud Server,  (add a space after period and a comma after Server)

File: content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md [hunk 8]
```diff
@@ -1,121 +1,109 @@
...
+2. Change the hostname to match the domain:
 
-Replace the current hostname (cloudserver1) with the one you need (in this case **mail.democloud.com**).
+    `sudo nano /etc/hostname`
 
-The second file to edit is:
+    Replace the current hostname `cloudserver1` with your selected domain name which in this example is **mail.democloud.com**.
 
-    sudo nano /etc/hosts
+3. The second file to edit can be opened for editing using the following command:
```
Comment (cyrichardson) [inline content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md]: You can open the second file for editing by using the following command:   (change passive to active)

File: content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md [hunk 9]
```diff
@@ -1,121 +1,109 @@
...
+    `hostname -f`
 
-and check the hostname:
+The output is now:
 
-    hostname -f
+    `mail.democloud.com`
 
-The output is now:
+**Note:** `mail.democloud.com` is an example hostname.
```
Comment (cyrichardson) [inline content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md]: is an example of a hostname

File: content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md [hunk 10]
```diff
@@ -1,121 +1,109 @@
...
 
-    mail.democloud.com
 
-Good start!
+### Change reverse DNS in the Control Panel
 
-### Configure reverse DNS
+Now, you must configure the rDNS on the server.
```
Comment (cyrichardson) [inline content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md]: rDNS or RDNS (which was used above - use the same throughout, whichever one is correct)

File: content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md [hunk 11]
```diff
@@ -1,121 +1,109 @@
...
 
-Now, we need to properly set the reverse DNS for the server.
+1. Navigate to your server's DNS tab and scroll down to the Reverse DNS
+section.
 
-#### Change reverse DNS in the Control Panel
+2. Change the relevant field to the domain name which you'd like
```
Comment (cyrichardson) [inline content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md]: domain that you want

File: content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md [hunk 12]
```diff
@@ -1,121 +1,109 @@
...
+in the rDNS record. See [DNS - Creating a Reverse DNS Record](/how-to/create-a-reverse-dns-record-0 "DNS - Creating a Reverse DNS Record") for detailed instructions.
 
-Navigate to your Server's DNS tab and scroll down to the Reverse DNS
-section. Change the relevant field to the domain name which you'd like
-in the rDNS record. (See [DNS - Creating a Reverse DNS Record](/how-to/create-a-reverse-dns-record-0 "DNS - Creating a Reverse DNS Record"))
+#### Verify reverse DNS changes
```
Comment (cyrichardson) [inline content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md]: Verify rDSN (or RDSN) changes   (use whichever of the acronyms is correct)

File: content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md [hunk 13]
```diff
@@ -1,121 +1,109 @@
...
 
-#### Check the reverse DNS
+The RDNS can take 24-48 hours to propagate. We recommend that you wait until
```
Comment (cyrichardson) [inline content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md]: 24 to 48 hours

File: content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md [hunk 14]
```diff
@@ -130,5 +118,4 @@ In this case, the output includes the correct answer:
 
 ### Summary
 
-Preparing the Cloud Server is a vital step in any mail setup - even if only sending mail 
-from your application to an administrator, it is very important to get the basics right.
+Even if only sending mail from your application to an administrator, it is critical that the Cloud Server is properly configured prior to configuring it as a mail server.
```
Comment (cyrichardson) [inline content/cloud-servers/preparing-the-cloud-server-to-be-a-mail-server.md]: Even if you are only sending mail from your application to an administrator, remember that  it is criical that the Cloud Server is properly configured prior to configuring it as a mail server.

**Pull request merged**